src/sdl.h

```diff
@@ -0,0 +1,44 @@
+#include <math.h>
+#include <SDL2/SDL.h>
+
+#define SCALE 20
+
+struct COLOR
+{
+  int r;
+  int g;
+  int b;
+  int a;
+};
+
+struct PIXEL
+{
+  int x;
+  int y;
+  struct COLOR color;
+};
+
+struct COLOR palette[16] = {
+  {.r =   0, .g =   0, .b =   0, .a = 255}, //"000000"
```

Comment:
Should this line represent what comment says?
Don't know but following what comment says
```c
{ .r = 0, .g = 0, .b = 0, .a = 0 }
```

Am I missing something else?

Reply:
.a stands for alpha and set the transparency, the hex in the comment is only for the color, since all color are without any transparency I didn't added to the hex